Bottom: ddf0ad0494ddeb70b2ae3a104c1225549a08396b
Top:    aae2b420642001307f8ff7864223bbea285abb9b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 00:01:45 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 7980090..97f92a2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
